python/main.py

```diff
@@ -1,17 +1,20 @@
 import os
 import logging
 import pathlib
-from fastapi import FastAPI, Form, HTTPException, Depends
+from fastapi import FastAPI, Form, HTTPException, File, UploadFile
 from fastapi.responses import FileResponse
 from fastapi.middleware.cors import CORSMiddleware
 import sqlite3
 from pydantic import BaseModel
 from contextlib import asynccontextmanager
+import json
+import hashlib
 
 
 # Define the path to the images & sqlite3 database
 images = pathlib.Path(__file__).parent.resolve() / "images"
 db = pathlib.Path(__file__).parent.resolve() / "db" / "mercari.sqlite3"
+JSON_DB = pathlib.Path(__file__).parent.resolve() / "items.json"
```

Comment:
Since `items.json` is a database, maybe we can keep it in the `db` folder?
```suggestion
JSON_DB = pathlib.Path(__file__).parent.resolve() / "db" / "items.json"
```